lib/collection.js

```diff
@@ -522,6 +518,24 @@ Collection.prototype.insertMany = function(docs, options, callback) {
   if (typeof options === 'function') (callback = options), (options = {});
   options = options ? Object.assign({}, options) : { ordered: true };
 
+  const insertManySchema = {
+    w: { type: ['number', 'string'] },
+    wtimeout: { type: 'number' },
+    j: { type: 'boolean' },
+    serializeFunctions: { type: 'boolean', default: this.s.serializeFunctions },
```

Comment:
sometimes the default to `serializeFunctions` is `this.s.serializeFunctions`, and sometimes there is no default at all (L629). We should be consistent here I think

Reply:
I only set `this.s.serializeFunctions` as the default for `serializeFunctions` when it was [already being set](https://github.com/mongodb/node-mongodb-native/pull/1859/files/b77b26ac4403832c6e2aebc9d42319521bd6cdee#diff-a3644fc6e558da2bde71112d24ec75baL538). Should I be setting it as the default when it wasn't previously being assigned?

Reply:
Like we just talked about, I'll add this work to the follow-up epic and we can handle it then so no behavior is changed with this work.